### PR TITLE
Improved setup instructions in contributing.md

### PR DESCRIPTION
**Pull request by xBalbinus**
## Motivation

Currently, the setup instructions in contributing.md is a red herring. Until the one-button local env setup is complete, there is no simple "yarn bootstrap" that will simply install all dependencies

## Solution

Detailing all steps to install dependencies and to build / test deploy contracts.

## PR Checklist

- [x] Added Tests (manually tested)
- [x] Updated Documentation

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -68,10 +68,19 @@ If needed, reference Foundry's [Troubleshooting Installation](https://github.com
...
 
 ```
-$ yarn
-$ yarn bootstrap
+$ yarn install
+$ yarn build
+$ yarn compile
 ```
 
+After that, go to the `configuration` folder found in the [rust repo](https://github.com/nomad-xyz/rust). Pull whichever config you want from there, and place that inside your config folder in `/packages/deploy/config`.
```
Comment (prestwich) [inline CONTRIBUTING.md:76]: please see comment here:
https://github.com/nomad-xyz/monorepo/pull/292/files#r875347767

Comment (xBalbinus) [reply CONTRIBUTING.md:76]: Thanks for the feedback! Will remove this.

Comment (xBalbinus) [reply CONTRIBUTING.md]: Removed all content regarding deploy & extraneous yarn compile.

**Review by yourbuddyconner — APPROVED** [review]
Great first contribution, thanks for the PR!

**Comment by prestwich** [comment]
Quick note, we require commits to be signed by a verified key. You can read docs on setting that up [here](https://docs.github.com/en/authentication/managing-commit-signature-verification/about-commit-signature-verification). 

If you'd like, I can squash your branch and sign this work myself while you get commit signing set up :)

**Comment by xBalbinus** [comment]
Yeah I think that sounds nice, if you wouldn’t mind / won’t be much trouble
(don’t want this to be a blocker while I set up commit signing). Thank you
so much!!!

**Review by prestwich — APPROVED** [review]

**Comment by prestwich** [comment]
cool, thanks for the contribution :)

**Pull request merged**